client/src/components/Friends.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+const Friends = (props) => {
+
+
+  const [allUsersList, setAllUsersList] = useState(['']);
+  const [friendsList, setFriendsList] = useState(props.friends);
+
+  const addFriend = (e) => {
+    let friends = [...friendsList, e.target.getAttribute('value')];
+    axios.post('/api/updateFriends', { friends })
+    setFriendsList(friends)
+  }
+  const removeFriend = (e) => {
+    let friends = friendsList.filter(location => location !== e.target.getAttribute('value'));
+    axios.post('/api/updateFriends', { friends })
+    setFriendsList(friends)
+  }
+  
+
+  const AllUsers =  () => {
```

Comment:
This is a misleading name for this component. It renders all users *except* for your friends.